File: cpp/uclnbcnn_.cpp
#include <iostream>
using namespace std;

int main () {
	int a,b,tich,r;
	cout << "Nhap a,b "; cin >> a >> b;
	tich=a*b;
	
	while (b!=0) {
		r = a % b;
		a = b;
		b = r;
	}
	
	cout << "UCLN: " << a << endl;
	cout << "BCNN: " << tich/a;
}

File: py/ucln_bcnn.py
a = int(input("Nhap a: "))
b = int(input("Nhap b: "))
luua = a
luub = b

while (a != b):
    if (a > b): a = a - b
    else: b = b - a
print("UCLN: ",a)
print("BCNN: ",luua * luub / a)